functions/src/firestore-types/lobby.ts

```diff
@@ -24,6 +24,11 @@ export type Player = {
    * the number of players that have voted for this player
    */
   votes?: number;
+
+  /**
+   * displays the role of the player
```

Comment:
should be a description of what it is not what it does. it's a variable it doesn't do anything